pygsti/report/workspaceplots.py

```diff
@@ -1899,6 +2075,33 @@ def _create(self, plottypes, circuits, dataset, model, prec, sum_up, box_labels,
 def _mx_fn_from_elements(plaq, x, y, extra):
     return plaq.elementvec_to_matrix(extra[0], extra[1], mergeop=extra[2])
 
+#modified version of the above meant for working with circuit lists
+def _mx_fn_from_elements_circuit_list(circuit_list, extra):
```

Comment:
I don't see a way around basically having this duplicate function, but what are your thoughts on including this as an elementvec_to_matrix function in CircuitList so that it is more "compatible" stylistically with how the plaq function is done?
I mostly want to avoid someone updating the elementvec_to_matrix in plaq and forgetting to do something similar here - I know that moving this into CircuitList would not enforce it, but at least a quick search for elementvec would have both classes pop up instead of remembering to change the second instance here.

Reply:
Just pushed a commit which does this refactor, and it looks like everything is still working with these changes given some simple tests.